sonar-php-plugin/src/test/java/org/sonar/plugins/php/PhpPluginTest.java

```diff
@@ -51,4 +55,28 @@ public void test_sonarlint() {
     assertThat(context.getExtensions()).hasSize(9);
   }
 
+  @Test
+  public void should_contain_REPORT_PATHS_after_6_2() throws Exception {
+    final Plugin.Context context6_2 = qubeContext(Version.create(6, 2));
+    final PhpPlugin plugin = new PhpPlugin();
+    plugin.define(context6_2);
+
+    assertThat(extensionKeysOf(context6_2)).contains(PhpPlugin.PHPUNIT_COVERAGE_REPORT_PATHS_KEY);
```

Comment:
Since I added the deprecation notices to the old properties I have a test checking for them, so it guarantees that those properties are still there in 6.2